hexrdgui/calibration/polarview.py

```diff
@@ -491,7 +491,6 @@ def reapply_masks(self):
         # Both "visible" and "boundary" masks are applied to the
         # computational image
         comp_img = self.apply_boundary_masks(img)
-        comp_img = self.apply_tth_distortion(comp_img)
```

Comment:
Yes and no - this commit appears completely out of sync with my local branch so I'm going to re-push those changes. The deletion is intentional but only because we're swapping the order of operations so we don't need to apply the tth distortion twice.